src/common.js

```diff
@@ -125,6 +130,7 @@ exports.getUserHome = getUserHome;
 //   parseOptions('-a', {'a':'alice', 'b':'bob'});
 // Returns {'reference': 'string-value', 'bob': false} when passed two dictionaries of the form:
 //   parseOptions({'-r': 'string-value'}, {'r':'reference', 'b':'bob'});
+// XXX: Is it sufficient to have this as part of wrap(), or should it be part of plugin.utils? (ie. do plugins need access to it directly?)
 function parseOptions(opt, map) {
```

Comment:
Unsure if plugins need access to this directly, or having it as part of `wrap` is sufficient.